cmd/simpleargs/README.md

```diff
@@ -0,0 +1,41 @@
+# simpleargs
```

Comment:
now that the tool does a lot more than just parsing names should name it differently
`commentMonitor` sound like a good name.